pkg/analysis/passes/metadatavalid/metadatavalid.go

```diff
@@ -54,6 +61,12 @@ func run(pass *analysis.Pass) (interface{}, error) {
 		return nil, err
 	}
 
+	_, err = os.Stat(metadataPath)
```

Comment:
before this would not even execute, now it executes and report a missing plugin.json